src/pages/Map.css

```diff
@@ -3,3 +3,12 @@
   height: 100vh;
   width: 100%;
 }
+
+.location-pin {
+  cursor: pointer;
+  transform: translate(-50%, -50%);
```

Comment:
Yessss!!! I forgot to bring this up last week, but for the pins to be directly over the places, you need to translate them back by half the pin width. Nice work! How did you realize this was needed?

Reply:
We did a lot of looking at github examples, stackover and trial & error trying to find how to get the location pins to be a click event with the google-map-react component. In the process we found an example with this!